<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VCSBase::BaseCheckoutWizardPage</class>
 <widget class="QWizardPage" name="VCSBase::BaseCheckoutWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="repositoryLabel"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="repositoryLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="checkoutDirectoryLabel">
       <property name="text">
        <string>Checkout Directory:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pathLabel">
       <property name="text">
        <string>Path:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="Utils::PathChooser" name="pathChooser"/>
     </item>
     <item row="1" column="1">
      <widget class="Utils::ProjectNameValidatingLineEdit" name="checkoutDirectoryLineEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::ProjectNameValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/projectnamevalidatinglineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
